packages/studio/src/components/Viewport/ViewportMenu.tsx

```diff
@@ -0,0 +1,51 @@
+import { useCallback } from "react";
+import { EXPANDED_VIEWPORTS, Viewport } from "./defaults";
+import useStudioStore from "../../store/useStudioStore";
+
+interface OptionProps extends Viewport {
+  closeMenu: () => void;
+}
+
+export default function ViewportMenu(props: {
+  closeMenu: () => void;
+}): JSX.Element {
+  return (
+    <div className="absolute z-20 rounded bg-white text-sm text-gray-700 shadow-lg flex flex-col items-start py-1">
+      {Object.values(EXPANDED_VIEWPORTS).map((val) => {
+        return <Option key={val.name} closeMenu={props.closeMenu} {...val} />;
+      })}
+    </div>
+  );
+}
+
+function Option(props: OptionProps) {
```

Comment:
Small nit, could we do `Option({ name, closeMenu, styles }: OptionProps)` here? It'd simplify some of the code below.